backend/app/Models/User.php

```diff
@@ -18,8 +18,11 @@ class User extends Authenticatable
      * @var string[]
      */
     protected $fillable = [
-        'name',
+        'fname',
+        'mname',
+        'lname',
         'email',
+        'role',
         'password',
     ];
```

Comment:
Tip: you can use `protected $guarded = [];` instead of this

Reply:
Hello sir what is this for `protected $guarded = [];`